src/main/java/fm/last/moji/impl/FileUploadOutputStream.java

```diff
@@ -90,12 +92,17 @@ public void close() throws IOException {
           delegate.close();
         } finally {
           try {
-            String message = httpConnection.getResponseMessage();
             int code = httpConnection.getResponseCode();
             if (HttpURLConnection.HTTP_OK != code && HttpURLConnection.HTTP_CREATED != code) {
-              throw new IOException(code + " " + message);
-            } else {
```

Comment:
This debug message when the response is OK has now been lost, it's probably not that important and easy to put back I'm just not sure it was your intention to remove this "happy day" scenario.